### ISSUE TITLE
YOLOv4 integration with the Intel RealSense camera

### ISSUE THREAD

**Issue by VAIBHAVPATEL97** [AlexeyAB/darknet#7995]
Hello,
Is it possible to use Intel RealSense camera for real-time camera input to the YOLOv4 model?. I am trying to achieve this idea on Jetson TX2 or Jetson AGX Xavier. I did some research on google but was unable to find any leads. Any suggestions or comments is appreciated. Thanks in advance.

**Comment by stephanecharette** [comment]
Is there anything super special about that camera that would prevent it from being used?

**Comment by VAIBHAVPATEL97** [comment]
This camera is capable of taking infrared, depth, and RGB pictures. I would like to use only RGB video to test the video but the YOLOv4 model chooses the Infrared monochrome video setting by default whenever I connect this camera. So, is there any python script that I can change within the repo so that I can get RGB video stream in real-time?

**Comment by VAIBHAVPATEL97** [comment]
Yeah I was able to figure out and get working now I want to change the resolution of every test run which I do. Any suggestion for that @stephanecharette. Thanks for your quick response.

**Comment by stephanecharette** [comment]
Suggestions for what?  For changing the camera resolution?  I have no idea, I've never even googled that camera.  Nor have you told us what you are using.  I assume OpenCV?  If so, it has well-documented API for changing the camera resolution.

**Comment by VAIBHAVPATEL97** [comment]
I am trying to use this camera (Intel RealSense Camera) for real-time detection on the YOLOv4 model. So, to compare the performance of the trained model on different resolutions I want to change the resolution of the video captured by the camera. There is a darknet_video.py file present in the darknet repository which I am trying to change the achieve a different resolution. So, my question @stephanecharette to you is am I going correct or there is any other python file present in the darknet repo which can be changed to achieve the above mentioned task.

**Comment by lizhiying-2019** [comment]
@VAIBHAVPATEL97 Hello, I want to use Intel RealSense camera for real-time camera input to the YOLOv5 model. Could you please tell me how to get the stream? I set the camera  as 0 when I connect my computer by USB3.0. But it is not working in Win10.

**Comment by VAIBHAVPATEL97** [comment]
@lizhiying-2019 You can try to change the port number to 2 or 3 depending on the corresponding port number on your PC. Did you check the port number to which you camera is connected to the PC?
